### DISCUSSION TITLE
Mixing ratio denominator units for tracers, wet ? dry ? something in between?

### DISCUSSION THREAD

**Discussion by aschuh** [ufs-community/ufs-weather-model#1881] [Q&A] [answered]
It isn't clear to me what the "mixing ratio" unit is for tracers for UFS.  The field_table just says "mixing ratio".  I've looked through old threads and it seems that a mixing ratio might be interpreted as tracer mass relative to "dry air mass + water vapor", is that correct?  So, it's really not "dry mix ratio" or "wet mix ratio" ?  I'm working w/ long lived tracers. I'm assuming if one multiplies tracer concentrations by delp, and sums up globally, which I believe is what is used in the fv_diagnostics.F90 tools to add up mass globally for conservation checks, then we are trying to conserve tracer mass relative to something "in between" wet and dry mass, is that correct?

**Comment by natalie-perlin** [comment, marked as answer]
@aschuh - 
mixing ratio units may depend on a tracer. For water-related mixing ratios, such as specific humidity, cloud/rain/ice/snow/graupel water mixing ratio, the units are kg/kg. Th units are specified in weather model field tables, i.e., under ./tests/parm/field_table/field_table_* .
This [link to UFS-WM Documentation (Chapter 4.2. Model Configuration Files. 4.2.2. field_table file)](https://ufs-weather-model.readthedocs.io/en/latest/InputsOutputs.html#field-table-file) has an example of the field_table entry, in a green rectangle.

I do see your concern about the denominator for conserving tracers. Theoretically, it is for "dry air", but model parameterization could use approximations. In this case, could you use different units, such as g/m^3, ug/m^3 (microgram per meter-cubed), or ppm/ppb? 
Couple of examples from  field_table_GOCART:
```
# diagnostic PM tracers
 "TRACER", "atmos_mod", "pm25"
           "longname",     "primary PM25 mixing ratio"
           "units",        "ug/m3"
       "tracer_usage", "chemistry", "type=diagnostic"
       "profile_type", "fixed", "surface_value=0.0" /
 "TRACER", "atmos_mod", "pm10"
           "longname",     "primary PM10 mixing ratio"
           "units",        "ug/m3"
       "tracer_usage", "chemistry", "type=diagnostic"
       "profile_type", "fixed", "surface_value=0.0" /
```

```
 "TRACER", "atmos_mod", "dms"
           "longname",     "DMS mixing ratio"
           "units",        "ppm"
       "tracer_usage", "chemistry"
       "profile_type", "fixed", "surface_value=0.0" /
 "TRACER", "atmos_mod", "msa"
           "longname",     "msa mixing ratio"
           "units",        "ppm"
       "tracer_usage", "chemistry"
       "profile_type", "fixed", "surface_value=0.0" /
```
